debug_tool/utils/db.py

```
@@ -98,23 +98,123 @@ def get_overview(self) -> Dict[str, Any]:
+    def _get_collections_via_sqlite(self) -> List[str]:
+        """绕过 chromadb API，直读 sqlite3 获取集合名（版本兼容回退）"""
+        try:
+            db_path = os.path.join(self.chromadb_path, "chroma.sqlite3")
+            if not os.path.exists(db_path):
+                return []
+            conn = sqlite3.connect(db_path)
+            conn.row_factory = sqlite3.Row
+            rows = conn.execute("SELECT name FROM collections").fetchall()
+            conn.close()
+            return [r["name"] for r in rows]
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)